test/jdk/java/awt/Canvas/MultiGraphicsTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 6616089
+ * @summary Display an image in all available GraphicsConfigurations
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual MultiGraphicsTest
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Canvas;
+import java.awt.Dimension;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.GraphicsConfiguration;
+import java.awt.GraphicsDevice;
+import java.awt.GraphicsEnvironment;
+import java.awt.Image;
+import java.awt.MediaTracker;
+import java.awt.Toolkit;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.net.URL;
+import java.util.List;
+
+public class MultiGraphicsTest extends Canvas {
+    final static String IMAGEFILE = "duke_404.gif";
+    static Image jim;
+    MediaTracker tracker;
+    int w, h;
+
+    public static void main(String[] args) throws Exception {
+        String INSTRUCTIONS = """
+                This test displays several Windows containing an image,
+                one Window for each available GraphicsConfiguration.
+                Depending on the GraphicsConfiguration, images may be
+                displayed in color or in grayscale and/or displayed at a
+                lower bitdepth.
+                The number of GraphicsConfigurations will be printed below
+                as the test is starting up.
+                On Solaris, making some Windows active will cause some of
+                the other Windows to display strangely. This is normal.
+                Ensure that there are as many Windows created as there are
+                available GraphicsConfigurations.
+                Examine each Window to ensure it displays Jim.
```

Comment:
Jim ? You mean Duke.

Also remove mention of Solaris, it hasn't been a supported platform for several years

Reply:
Updated.